src/backend.js

```diff
@@ -106,6 +97,18 @@ var _ = Mavo.Backend = $.Class({
 			call.searchParams.set("timestamp", Date.now());
 		}
 
+		if ($.type(req.data) === "object") {
+			if (req.method == "GET") {
+				for (const p in req.data) {
+					call.searchParams.set(p, req.data[p]);
+				}
+				req.data = "";
```

Comment:
`undefined` caused issues?

In general, have you verified that the tests that were passing before this change still pass?

Reply:
All the tests still pass. I think it was caused by another issue, and `undefined` was probably a false cause.

Reply:
I'd just do `delete req.data` which emulates never setting it in the first place (since we've incorporated it to the URL, we don't need it anymore as a separate parameter)